packages/calcite-components/src/components/input-time-picker/input-time-picker.e2e.ts

```diff
@@ -665,11 +665,10 @@ describe("calcite-input-time-picker", () => {
     expect(await getInputValue(page)).toBe("٠٢:٣٠:٢٥ م");
 
     inputTimePicker.setProperty("lang", "zh-HK");
-    inputTimePicker.setProperty("numberingSystem", "hanidec");
```

Comment:
I think this needs to be

```ts
inputTimePicker.setProperty("numberingSystem", "latn");
```

to match the expectation

Reply:
Good catch. Fixing!